db/schema.rb

```diff
@@ -197,14 +197,4 @@
   add_index "votes", ["user_id"], name: "index_votes_on_user_id", using: :btree
   add_index "votes", ["votation_id"], name: "index_votes_on_votation_id", using: :btree
 
-  create_table "workshops_engine_workshops", force: true do |t|
```

Comment:
Those changes appeared after run the migrations on my local. Somehow the schema.rb wasn't updated.